### ISSUE TITLE
RMCFG in maui.cfg

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#185]
**Reported by anonymous on 23 Jun 2006 13:11 UTC**
It appears that RMCFG[] in maui.cfg always included the hostname of the buildhost
could this be changed to something less confusing...

**Comment by dikim33** [comment]
**Comment by bli on 24 Jun 2006 04:59 UTC**
I added the following line to the spec file:

```
perl -pi -e "s~RMCFG\[.*\]~RMCFG[$HOSTNAME]~" %{maui_prefix}/maui.cfg
```

**Comment by dikim33** [comment]
**Comment by bli on 25 Jun 2006 05:35 UTC**
(In [Rebuilt MAUI RPM
- added init scripts for SUSE Linux
- fixes #186 regarding RMCFG[](5044]))
- hardcodes "root" to be ADMIN1 user (bug discovered by Fernando)
- please rebuild on all other distributions!
